pom.xml

```
@@ -36,7 +36,7 @@
         <github.slug>okta/okta-spring-boot</github.slug>
         <okta.sdk.version>6.0.0</okta.sdk.version>
         <okta.commons.version>1.2.8</okta.commons.version>
-        <embedded.tomcat.version>9.0.53</embedded.tomcat.version>
+        <embedded.tomcat.version>10.0.12</embedded.tomcat.version>
```

```suggestion
        <embedded.tomcat.version>9.0.54</embedded.tomcat.version>
```
I'd strongly suggest staying on `9.x` until Spring Boot moves to `10.x`. (`9.0.54` is the current version defined in the spring boot BOM: https://search.maven.org/artifact/org.springframework.boot/spring-boot-dependencies/2.5.6/pom)

---

yeah, I realized that and hence closed the PR.